setup.py

```diff
@@ -53,6 +53,7 @@
       classifiers=[
           'Framework :: Plone',
           'Framework :: Plone :: 4.2',
```

Comment:
✂️ I'd remove that because there is no test-buildout for 4.2, so we are not actively supporting it. It might work, but that is not relevant then.

Reply:
👍